### PR TITLE
Replace npm deploy key

### LINKED ISSUE #2105: Replace NPM token
Tokens have been revoked and need re-issuing.

These tokens were revoked because of a backdoor in eslint-scope@3.7.2 that stole the content of ~/.npmrc: https://status.npmjs.org/incidents/dn7c1fgrr7ng

This affected any installation of eslint-scope@3.7.2 for two hours last thursday (12 July 2018). For the exact timeline, see https://eslint.org/blog/2018/07/postmortem-for-malicious-package-publishes

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #2105

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-  - node bin/build-locales
-  - npm run build
-  - npm run lint
-  # run integration tests using an addons-linter binary in a production-like environment
-  - npm run test-integration:production
+- npm run test-ci
+- node bin/build-locales
+- npm run build
+- npm run lint
+- npm run test-integration:production
```
Comment (EnTeQuAk) [inline .travis.yml]: Why did you change the indentation and remove the comment? Or is that maybe what the `travis` cli did?

Comment (muffinresearch) [reply .travis.yml]: Travis CLI does that.

Comment (muffinresearch) [reply .travis.yml]: I'll fix it.
